_includes/pbc/all/install-features/202304.0/install-the-picker-user-login-feature.md

```diff
@@ -0,0 +1,286 @@
+| WarehouseTokenAuthorizationStrategyPlugin                 | Checks if if the request identity is valid user and warehouse.                                                                                                                                                     |               | Spryker\Zed\OauthWarehouse\Communication\Plugin\Authorization                                 |
+| OauthWarehouseInstallerPlugin                             | Installs warehouse OAuth scope data.                                                                                                                                                                               |               | Spryker\Zed\OauthWarehouse\Communication\Plugin\Installer                                     |
+| WarehouseOauthUserProviderPlugin                          | Retrieves warehouse user if `OauthUserTransfer.idWarehouse` provided & expands the `OauthUserTransfer` if warehouse user exists.                                                                                   |               | Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth                                         |
+| WarehouseOauthScopeProviderPlugin                         | Checks whether the grant type is \Spryker\Zed\OauthWarehouse\OauthWarehouseConfig::WAREHOUSE_GRANT_TYPE.                                                                                                           |               | Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth                                         |
+| WarehouseOauthRequestGrantTypeConfigurationProviderPlugin | Checks whether the requested OAuth grant type equals to \Spryker\Zed\OauthWarehouse\OauthWarehouseConfig::WAREHOUSE_GRANT_TYPE and whether the requested application context equals to GlueBackendApiApplication.  |               | Spryker\Zed\OauthWarehouse\Communication\Plugin\Oauth                                         |
+
+**src/Pyz/Glue/GlueBackendApiApplication/GlueBackendApiApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Glue\GlueBackendApiApplication;
+
+use Spryker\Glue\GlueBackendApiApplication\GlueBackendApiApplicationDependencyProvider as SprykerGlueBackendApiApplicationDependencyProvider;
+use Spryker\Glue\WarehouseOauthBackendApi\Plugin\GlueBackendApiApplication\WarehouseRequestBuilderPlugin;
+use Spryker\Glue\WarehouseOauthBackendApi\Plugin\GlueBackendApiApplication\WarehouseRequestValidatorPlugin;
+use Spryker\Glue\WarehouseOauthBackendApi\Plugin\GlueBackendApiApplication\WarehouseTokensBackendResourcePlugin;
+
+class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiApplicationDependencyProvider
```

Comment:
* The original integration uses the new use Spryker\Glue\OauthBackendApi\Plugin\GlueBackendApiApplication\UserRequestBuilderPlugin plugin.